pkg/k8/client/client.go

```diff
@@ -4,17 +4,25 @@ import (
 	"os"
 	"path"
 
+	log "github.com/sirupsen/logrus"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 )
 
-//Get returns a kubernetes client. If namespace is empty, it will use the default namespace configured.
-func Get(namespace string) (string, *kubernetes.Clientset, *rest.Config, error) {
-	home := os.Getenv("HOME")
+//Get returns a kubernetes client.
+// If namespace is empty, it will use the default namespace configured.
+// If path is empty, it will use the default path configuration
+func Get(namespace, configPath string) (string, *kubernetes.Clientset, *rest.Config, error) {
+	if configPath == "" {
+		configPath = path.Join(os.Getenv("HOME"), ".kube/config")
```

Comment:
I would set this value from the caller:
https://github.com/okteto/cnd/pull/146/files#diff-ff7686b39bf90dc2520886fb874371a4R90